src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  padding: 0;
+  list-style: none;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background: url(../images/star.svg) no-repeat center;
+  }
+
+  &--5 &__star {
+    margin-bottom: 0;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
```

Comment:
`margin: 0`?

Reply:
@VolodymyrKirichenko I realized that `margin-bottom: 0;` is useless here. Is something else wrong?

Reply:
@Valeraom 
I think you didn't understand my comment)
you can use `margin: 0` for the last child, which will be shorter and more readable.
You didn't make any mistake, it's just an improvement of your code